store-implementation/proxy-store/src/main/java/uk/gov/gchq/gaffer/proxystore/ProxyStore.java

```diff
@@ -140,7 +154,10 @@ public Set<Class<? extends Operation>> getSupportedOperations() {
 
     @Override
     public boolean isSupported(final Class<? extends Operation> operationClass) {
-        return getSupportedOperations().contains(operationClass);
+        return AddNamedView.class.isAssignableFrom(operationClass)
+                || AddNamedOperation.class.isAssignableFrom(operationClass)
+                ? super.getSupportedOperations().contains(operationClass)
+                : getSupportedOperations().contains(operationClass);
```

Comment:
I think this would be easier to understand/read if the ternary operator assigned to a variable first. Could be the same number of lines.

Reply:
I disagree, if the line start with `||` `&&` `?` `:` I think it reads well.

If I added a variable I would need to call it "isClassAddNamedViewOrAddNamedOp" which is verbose for such a small method.

Reply:
```java
  final boolean isClassAddNamedViewOrAddNamedOperation = AddNamedView.class.isAssignableFrom(operationClass) || AddNamedOperation.class.isAssignableFrom(operationClass);
        return isClassAddNamedViewOrAddNamedOperation
                ? super.getSupportedOperations().contains(operationClass)
                : getSupportedOperations().contains(operationClass); 
```